src/components/siteFeatures/index.tsx

```diff
@@ -41,7 +43,7 @@ const SiteFeatures: React.FC<SiteFeaturesProps> = ({
     case true:
```

Comment:
❓ Do we know why whoever originally wrote this code used a switch statement here? It looks like it could be simplified to an if statement or JavaScript's [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

Reply:
not sure, I guess an argument could be made for readability

Reply:
I would even argue using the switch is _less_ readable but 🤷

Reply:
yeah, at first I thought there was some case for if editingFeatures was undefined, or something else unexpected